KenticoCloud.Delivery/DeliveryClient.cs

```diff
@@ -522,23 +522,22 @@ public DeliveryClient(string projectId, string previewApiKey) : this(new Deliver
                 throw new InvalidOperationException("Preview API and secured Delivery API must not be configured at the same time.");
             }
 
-            return _deliveryOptions.EnableResilienceLogic ? await GetRepeatableDeliverResponseAsync(endpointUrl) : await GetSolitaryDeliveryResponseAsync(endpointUrl);
-        }
-
-        private async Task<JObject> GetRepeatableDeliverResponseAsync(string endpointUrl)
-        {
-            var policyResult = await ResiliencePolicyProvider?.Policy?.ExecuteAndCaptureAsync(() =>
-                {
-                    return SendHttpMessage(endpointUrl);
-                }
-            );
-
-            return await GetResponseContent(policyResult?.FinalHandledResult ?? policyResult?.Result);
-        }
-
-        private async Task<JObject> GetSolitaryDeliveryResponseAsync(string endpointUrl)
-        {
-            return await GetResponseContent(await SendHttpMessage(endpointUrl));
+            if (_deliveryOptions.EnableResilienceLogic)
```

Comment:
why do I see retry logic-related code in this PR? is that intentional?

Reply:
Yes, it is. The PR goes from the 100-unresolvable-references branch to the 100-retry-policy branch. The changes are still closely related to the retry policy.